lineoperations/src/lo_prefs.c

```diff
@@ -117,7 +117,7 @@ lo_init_prefs(GeanyPlugin *plugin)
 	lo_info->use_collation_compare = utils_get_setting_boolean(config,
 		"general", "use_collation_compare", FALSE);
 
-	printf("VALUE: %d\n", lo_info->use_collation_compare);
+	/* printf("VALUE: %d\n", lo_info->use_collation_compare); */
```

Comment:
I would drop it completely. There is no point in having commented debug flying around. If needed, it can be restored from the GIT history.